builder/azure/arm/step_capture_image.go

```diff
@@ -87,11 +85,19 @@ func (s *StepCaptureImage) Run(ctx context.Context, state multistep.StateBag) mu
 
 	if err == nil {
 		if isManagedImage {
+			s.say("Capturing image ...")
+			var targetManagedImageResourceGroupName = state.Get(constants.ArmManagedImageResourceGroupName).(string)
+			var targetManagedImageName = state.Get(constants.ArmManagedImageName).(string)
+			var targetManagedImageLocation = state.Get(constants.ArmLocation).(string)
 			s.say(fmt.Sprintf(" -> Image ResourceGroupName   : '%s'", targetManagedImageResourceGroupName))
 			s.say(fmt.Sprintf(" -> Image Name                : '%s'", targetManagedImageName))
 			s.say(fmt.Sprintf(" -> Image Location            : '%s'", targetManagedImageLocation))
 			err = s.captureManagedImage(ctx, targetManagedImageResourceGroupName, targetManagedImageName, imageParameters)
+		} else if isSIGImage {
+			// It's possible to create SIG image
+			return multistep.ActionContinue
 		} else {
+			s.say("Capturing image ...")
```

Comment:
I've just moved it here from line 69

Reply:
yeah earlier it was at the top and thus it was correct as it could capture image in the if block. Now imo it makes more sense as line 88 tells its capturing image and this one is better suited to say capturing vhd. but not a big deal.